nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -476,6 +477,12 @@ in
           elisa
           gwenview
           okular
+
+          # For kinfocenter
+          pkgs.glxinfo
+          pkgs.pciutils
+          pkgs.vulkan-tools
+          pkgs.xorg.xdpyinfo
```

Comment:
Now that I think about it, I could instead wrap the executable of pkgs/desktops/plasma-5/kinfocenter.nix to include these in PATH...

Reply:
🤔 yeah thats a better idea. Lets do that.

Reply:
Do I do it in this PR or in another PR?